src/include/network/lan/hardware.rb

```diff
@@ -240,24 +240,8 @@ def initHardware
 
       Builtins.y2milestone("hotplug=%1", LanItems.hotplug)
 
-      Ops.set(
-        @hardware,
-        "devices",
-        LanItems.FreeDevices(Ops.get_string(@hardware, "realtype", ""))
-      ) # TODO: id-, bus-, ... here
-      if !Builtins.contains(
-        Ops.get_list(@hardware, "devices", []),
-        Ops.get_string(@hardware, "device", "")
-      )
-        Ops.set(
-          @hardware,
-          "devices",
-          Builtins.prepend(
-            Ops.get_list(@hardware, "devices", []),
-            Ops.get_string(@hardware, "device", "")
-          )
-        )
-      end
+      # list of free device names when e.g. adding new device
+      @hardware["devices"] = LanItems.new_type_devices(@hardware["realtype"], 10)
```

Comment:
what is that magic `10` ? I really love to see some meaningful constant there or at least explanation in comment.

Reply:
I'd like to ... someone decided in the past ;-)

Reply:
constant named